### PR TITLE
91 Service flow test results based on Ruuter logs

### PR DESCRIPTION
**Pull request by baha-a**

### CODE REVIEW

**Review by Minwasko — APPROVED** [review]

File: DSL/Liquibase/changelog/1682662304-ruuter-requests-logs.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+-- liquibase formatted sql
+-- changeset baha-a:1682662304
+CREATE TABLE requestLogs (
```
Comment (trevorling) [inline DSL/Liquibase/changelog/1682662304-ruuter-requests-logs.sql:3]: Please rename: `request_logs`

**Review by trevorling — CHANGES_REQUESTED** [review]

File: DSL/Ruuter/POST/generate-document.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+getUserInputByChatId:
+  call: http.post
+  args:
+    url: https://byk-resql:8443/get-user-inputs
+    body:
+      chatId: ${chatId}
+  result: getUserInputResult
+
+fromatUserInput:
```
Comment (trevorling) [inline DSL/Ruuter/POST/generate-document.yml:15]: Typo: `fromat` -> `format`

File: DSL/Ruuter/POST/generate-document.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  result: formatedUserInput
+
+convertDataToPDF:
+  call: http.get
+  args:
+    url: http://data_mapper:3005/js/convert/pdf
+    body:
+      data: ${formatedUserInput}
+  result: pdfFile
+  next: returnSuccess
```
Comment (trevorling) [inline DSL/Ruuter/POST/generate-document.yml:31]: Http.get and body has data?!

**Review by trevorling — APPROVED** [review]

**Review by 1AhmedYasser — APPROVED** [review]

File: DSL/Liquibase/changelog/1682662304-ruuter-requests-logs.sql [hunk 4]
```diff
@@ -0,0 +1,9 @@
+-- liquibase formatted sql
+-- changeset baha-a:1682662304
+CREATE TABLE request_logs (
+  id BIGSERIAL PRIMARY KEY,
+  timestamp TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+  level TEXT NOT NULL,
+  statusCode INT NOT NULL DEFAULT 0,
+  content TIMESTAMPTZ NOT NULL DEFAULT NOW(),
```
Comment (trevorling) [inline DSL/Liquibase/changelog/1682662304-ruuter-requests-logs.sql:8]: `,` comma is redundant here!

**Comment by trevorling** [comment]
@baha-a 
One minor fix what came out!

**Pull request merged**